CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contribution guidelines
+
+We love and welcome contributions!
+
+In order to maintain a high quality, we run a great number of checks on
+different OS / Compiler combinations, and using different analysis tools.
+
+## Prerequisites
+
+Building and testing `sentry-native` currently requires the following tools:
+
+- **CMake** and a supported C/C++ compiler, to actually build the code.
+- **python** and **pytest**, to run integration tests.
+- **clang-format** and **black**, to format the C/C++ and python code respectively.
```

Comment:
`black` and `pytest` should be installed in the virtualenv. Do we need to call them out here?